Dockerfile

```diff
@@ -178,8 +178,13 @@ RUN set -x \
 	&& git clone https://github.com/BurntSushi/toml.git /go/src/github.com/BurntSushi/toml \
 	&& (cd /go/src/github.com/BurntSushi/toml && git checkout -q $TOMLV_COMMIT)
 
+# Download godep & jq
+RUN go get github.com/tools/godep
+RUN curl -sSL http://stedolan.github.io/jq/download/linux64/jq -o /usr/local/bin/jq \
+   && chmod +x /usr/local/bin/jq
+
 # copy vendor/ because go-md2man needs golang.org/x/net
-COPY vendor /go/src/github.com/docker/docker/vendor
+RUN go get code.google.com/p/go.net/html
```

Comment:
This is a very old version of the dependency. Should be something like `golang.org/x/net/html`.